pom.xml

```diff
@@ -147,6 +147,8 @@
         <maven-jar-plugin.version>3.1.0</maven-jar-plugin.version>
         <maven-war-plugin.version>3.1.0</maven-war-plugin.version>
         <maven-shade-plugin.version>3.2.1</maven-shade-plugin.version>
+        
```

Comment:
Did you mean to add a blank line?

Reply:
yes, I was adding a separator between the version properties and a path property